src/pages/user-profile-view/user-profile-view.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { clone } from 'lodash'
+import Auth0LoginZone from '../../components/auth0-login-zone'
+import ReactTable from 'react-table'
+
+import './user-profile-view.scss'
+import LoadingElement from '../../components/generic-elements/loading-element'
+import ErrorComponent from '../../components/generic-elements/error-component'
+import moment from 'moment'
+
+const UserProfileView = (props) => {
+  const {
+    visualizations,
+    getUserVisualizations,
+    auth: { isAuthenticated },
+    loading,
+    loadFailure,
+    loadSuccess
+  } = props
+
+  useEffect(() => {
+    getUserVisualizations()
+  }, [])
+
+
+  const requestHasNotRun = !loadFailure && !loadSuccess
+  if (loading || requestHasNotRun) {
+    return (
+      <user-profile-view>
+        <LoadingElement />
+      </user-profile-view>
+    )
+  }
+
+  if (!isAuthenticated) {
+    return <ErrorComponent errorText={"You must be signed in to see your saved visualizations"} />
+  }
+
+
+  const visualizationsWithTitle = visualizations.map((visualization) => {
+    return {
+      ...visualization,
+      title: (<Link to={`/visualization/${visualization.id}`}>{visualization.title}</Link>),
+      created: moment.utc(visualization.created).local().format("YYYY-MM-DDTHH:mm:ss[Z]"),
+      updated: moment.utc(visualization.updated).local().format("YYYY-MM-DDTHH:mm:ss[Z]")
+    }
+  })
```

Comment:
Consider moving some/all of this transformation into a selector?  Either that or transform the data before we shove it into state.  Building this map here will happen on every render of this component.

Reply:
I originally had this in the selector but the `Link`s weren't actually routing to the `UserProfile` page. Would it be worth splitting up the time conversions and adding the `Link`s if we have to keep one in the view?

Reply:
Never mind. It ended up working by converting in the reducer before putting it in the store